src/components/core/footer/index.tsx

```diff
@@ -1,55 +1,59 @@
-import React from "react";
-import Link from "next/link";
-import Image from "next/image";
+import React from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
 
 export default function Footer({ isFixed }: { isFixed?: boolean }) {
   const generalList = [
+    { id: 1, title: 'Register', slug: '/register' },
+    { id: 2, title: 'Login', slug: '/login' },
+    { id: 3, title: 'Privacy Policy', slug: '/privacy' },
+    { id: 4, title: 'Terms & Conditions', slug: '/terms' },
+  ];
+
+  const helpSupport = [
+    { id: 1, title: 'Help Center', slug: '/contact' },
+    { id: 2, title: 'Contribution Guidelines', slug: '/contribute' },
+  ];
+
+  const socialIcons = [
```

Comment:
As mentioned in the issue that there is no need for icons as there is no official social media accounts of **Palettegram** other than its discord server, if you want to include, add just discord icon